cpp/journal_timeliness_checker.cc

```diff
@@ -106,8 +106,8 @@ int Main(int argc, char *argv[]) {
 
     if (not tardy_list.empty()) {
         if (EmailSender::SendEmail(sender_email_address, notification_email_address, "Überfällige Zeitschriften",
-                                   "Letzte Lieferung ans BSZ\n" + tardy_list, EmailSender::HIGH) > 299)
-            LOG_ERROR("failed to send email notofication!");
+                                   "Letzte Lieferung ans BSZ\n" + tardy_list) > 299)
```

Comment:
Why did you drop the priority?

Reply:
Because all other mails stick to the following scheme:
- High: System failures (e.g. pipeline crashes)
- Normal: Warnings
- Low: Successful status reports

Since this email contains only warnings, i tried to stick to normal priority.
(Also this tool is likely to be deprecated soon, so no point in sending high priority mails every day if everybody ignores it)